library/core/src/array/mod.rs

```diff
@@ -389,6 +389,23 @@ array_impl_default! {32, T T T T T T T T T T T T T T T T T T T T T T T T T T T T
 
 #[lang = "array"]
 impl<T, const N: usize> [T; N] {
+    /// Returns the number of elements in the array.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// let a: [usize; 3] = [1, 2, 3];
+    /// assert_eq!(a.len(), 3);
+    /// ```
+    #[cfg(not(bootstrap))]
+    #[doc(alias = "length")]
+    #[stable(feature = "rust1", since = "1.0.0")]
+    #[rustc_const_stable(feature = "array_len", since = "1.53.0")]
```

Comment:
I think the `#[stable]` feature and version should be the same as the one for `#[rustc_const_stable]`?